### PR TITLE
Update Migros

### PR DESCRIPTION
**Pull request by luschmar**

### CODE REVIEW

File: entries/m/migros.ch.json [hunk 1]
```diff
@@ -4,14 +4,19 @@
     "url": "https://www.migros.ch",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "custom-software"
     ],
```
Comment (hkamran80) [inline entries/m/migros.ch.json:8]:
```suggestion
    ],
    "custom-software": [
      "Migros Access app"
    ],
```

File: entries/m/migros.ch.json [hunk 2]
```diff
@@ -4,14 +4,19 @@
     "url": "https://www.migros.ch",
```
Comment (hkamran80) [inline entries/m/migros.ch.json]:
```suggestion
```

File: entries/m/migros.ch.json [hunk 3]
```diff
@@ -4,14 +4,19 @@
...
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "custom-software"
     ],
     "documentation": "https://login.migros.ch/help/security/2fa?hl=en",
```
Comment (hkamran80) [inline entries/m/migros.ch.json]:
```suggestion
    "documentation": "https://account.migros.ch/help#secureWhy2FactorPassword",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/m/migros.ch.json [hunk 4]
```diff
@@ -1,17 +1,25 @@
 {
   "Migros": {
     "domain": "migros.ch",
-    "url": "https://www.migros.ch",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "custom-software"
```
Comment (hkamran80) [inline entries/m/migros.ch.json:7]:
```suggestion
      "custom-software",
      "u2f"
```

If you support FIDO2/U2F as a second-factor in addition to passkeys, the `u2f` element should be added. Do you have a documentation link for it?

Comment (luschmar) [reply entries/m/migros.ch.json:7]: Sorry no. We don't have more documentation (just [Migros Account Help](https://account.migros.ch/help) ). We belive that the term passkey should be suffisant. In our help page, we point out, that the password can still be used -> https://account.migros.ch/help#securePasskey

**Comment by hkamran80** [comment]
I'll review this once more after the documentation page is working again.

**Comment by luschmar** [comment]
> I'll review this once more after the documentation page is working again.

Documentation is working again.

File: entries/m/migros.ch.json [hunk 5]
```diff
@@ -1,17 +1,26 @@
 {
   "Migros": {
     "domain": "migros.ch",
-    "url": "https://www.migros.ch",
     "tfa": [
       "sms",
-      "totp"
+      "totp",
+      "custom-software",
+      "u2f"
```
Comment (hkamran80) [inline entries/m/migros.ch.json]:
```suggestion
```

I just tried it and two-factor with a FIDO2 key is not implemented. Migros only has passkey support, not FIDO2 MFA.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to [server and browser caching](https://github.com/2factorauth/twofactorauth/wiki/FAQ-Caching).

**Pull request merged**